packages/vote/vote_context.go

```diff
@@ -16,7 +16,7 @@ type ObjectType uint8
 
 const (
 	// ConflictType defines an object type conflict.
-	ConflictType = iota
+	ConflictType ObjectType = iota
```

Comment:
why is this needed now - did it not work before?

Reply:
Before those values were not used when triggering the FPC vote as it was hardcoded to always use ConflictType. Now that we need to parametrize that there were problems with the typing.